CODE_OF_CONDUCT.md

```diff
@@ -1,128 +1,76 @@
-# Contributor Covenant Code of Conduct
+# Definity Contributor Code of Conduct
+## Preamble
+This code of conduct outlines expectations for participation in Definity-managed communities, as well as steps for reporting unacceptable behavior. People violating this code of conduct may be banned from the community.
 
 ## Our Pledge
-
-We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
-and orientation.
-
-We pledge to act and interact in ways that contribute to an open, welcoming,
-diverse, inclusive, and healthy community.
+We as members, contributors, and leaders pledge to make participation in our community an open, welcoming, inspiring, diverse, inclusive, healthy, and harassment-free community for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation. We pledge to act, interact, and provide in ways that contribute to it.
```

Comment:
After re-reading, I am in agreement with @TimeCipher, this just feels like a list of diversifying features & a list of "happy"-related buzz words. It should be changed to this, or with some relation to this. 
```suggestion
We as members, contributors, and leaders pledge to make participation in our community a welcoming and harassment-free place for everyone, regardless of age, body size, visible or invisible disability, or any other diversifying features. We pledge to act, interact, and provide in ways that contribute to it.
```

Reply:
I can see the buzzwords part, but the features are specifically listed because if it's too vague, anything can fly there. You could argue a *lot* of things as being a "diversifying feature".

Reply:
That is true, it really depends on if you want consistency with the README when deciding between using specifically my change or not. It should at least be shortened before merge.

Reply:
[Draft 5](https://github.com/DefinityTeam/.github/pull/2/commits/c295aa7a579d2d52a84f89b25ccb3321e27fde2f) resolves this. This is now concise and easier to read through.